### ISSUE TITLE
no Menu items or icons, sidebar and bottom bar vanishes

### ISSUE THREAD

**Issue by rajkotraja** [ijoomer-advance/ijoomer-adv-joomla#86]
This installation of component works well on server side, without any error but creates problems on App client.
Menus dont load on app, bottom bar vanishes and side bar comes blank, without options or configured tabs.

please solve this error.

**Comment by ijoomer-dev** [comment]
Related to https://github.com/ijoomer-advance/ijoomer-adv-joomla/issues/75

**Comment by rajkotraja** [comment]
So you mean to say is this solved?
If yes, please explain me how to fix it, or do i need to install new version that is 1.5.1 for joomla 3.X.
(i have joomla 2.5)
Thank you for your quick response.

**Comment by ijoomer-dev** [comment]
No it's not solved yet, it's in track and will be released in next released version soon or we will try to provide hot fix.

**Comment by rajkotraja** [comment]
Ok thank you,
Please try to provide hot fix as i am stuck with this, i was using older version before, but as we migrated to new server we changed to 1.5
So our release is stuck. so pls see if you can help us. Thank you

**Comment by ijoomer-dev** [comment]
@rajkotraja I notice that you are using Joomla 2.5.x 
If so please use our old version https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5forj2.5.x
As we are no longer support Joomla 2.5.x anymore because joomla doesn't. You must upgrade your site to Joomla 3.4 for many reasons. 

Ask if you need more info.

**Comment by rajkotraja** [comment]
I am currently using that only(https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5forj2.5.x), i am getting this error of no menus in that only.
i dont know why but joomla 3 is taking very much time to load, that is it takes time to open the application (Your splash screen stays for almost 30-40 seconds and then my home screen or menu screen loads that i have specified as default). Thats why we choose to stay on 2.5 with 1.5 version of your component and app.

Thank you

**Comment by ijoomer-dev** [comment]
Thanks for sepcification.
Are you getting error on http://jcms25.dev/administrator/index.php?option=com_ijoomeradv&view=menus ?

**Comment by rajkotraja** [comment]
Yeah, i m getting a error. if i install (https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5) or ver 1.5.1 designed for joomla 3.

If i check development mode for error reporting, this is the error i see:
Fatal error: Cannot redeclare class JDispatcher in /home/oraculus/public_html/oraculous/libraries/joomla/event/dispatcher.php on line 279‏

**Comment by ijoomer-dev** [comment]
Please keep joomla debug mode off and try to install the extension. It will work like that.

**Comment by rajkotraja** [comment]
Which version to install on which version of joomla you telling me?
Thank you

**Comment by ijoomer-dev** [comment]
Latest version: https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5.1
Any old version and Joomla 2.5 is not supported anymore.

**Comment by rajkotraja** [comment]
Ok, i tried this v1.5.1 on joomla 3.
it seems to be working well, pls tell me which is latest version of jom social extension.
i installed ext for jom3.2 for v1.5

and after installation i see:
Warning
Extensions Install: Version not supported. Please check extension version.

**Comment by rajkotraja** [comment]
well, i dont know why but its taking time to load the default page, it stays on on splash screen for very long on joomla 3.0, while this is quick on joomla 2.5
any fix for this?
Thank you

**Comment by ijoomer-dev** [comment]
Thanks for reporting, added separate issue to get into track, we will look into it.

**Comment by rajkotraja** [comment]
Thank you for opening this ticket, you can check this website once in your own ijoomer app.
It takes too long in that also: http://oraculous.in/sidraja/
Just try with this URL, it takes long time.

**Comment by ijoomer-dev** [comment]
For extensions you can go through https://github.com/ijoomer-advance/extensions/tree/develop/src to find the latest version based on your need.

**Comment by rajkotraja** [comment]
Can you provide any hot fix for no menu items error occurring for v1.5 for joomla 2.5
(https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5forj2.5.x)
i m stuck with this. can u solve this error for time being.

Thank you

**Comment by rajkotraja** [comment]
@ijoomer-dev 
Can you provide any hot fix for no menu items error occurring for v1.5 for joomla 2.5
(https://github.com/ijoomer-advance/ijoomer-adv-joomla/releases/tag/v1.5forj2.5.x)
i m stuck with this. can u solve this error for time being.

Thank you
